### ISSUE TITLE
midian don't shows the first midi track

### ISSUE THREAD

**Issue by bemtec** [jamiefaye/downrush#53]
In all my tests with different midi files, midian forgets to show the first midi track in a file

[MidiTest.zip](https://github.com/jamiefaye/downrush/files/4702941/MidiTest.zip)

**Comment by jamiefaye** [comment]
You are correct, of course. Funny thing is that a single track Midi file works OK, but a 4 track Midi file only shows 3 tracks. An "Off by 1" error, perhaps.

**Comment by bemtec** [comment]
Thank you for the reply and thank you for your fantastic work!
